lablib/processors/ayon_hiero_effect_file.py

```diff
@@ -43,7 +58,11 @@ def repo_operators(self) -> Dict:
         """List of repositioning operators to be processed."""
         return self._repo_ops
 
-    def _load(self) -> None:
+    def load(self) -> None:
```

Comment:
this could deserve a docstring now that this function is not marked private anymore :)

let's just re-use the one from the old `load()`:

```
"""Loads the effects file.
        Attention:
            This method clears the lists of all operators before loading.
"""
```